nservicebus/sagas/saga-not-found.md

```diff
@@ -13,10 +13,12 @@ snippet: saga-not-found
 
 Note that in the example above the message will be considered successfully processed and sent to the audit queue even if no saga was found. Throw an exception from the `IHandleSagaNotFound` implementation to cause the message to end up in the error queue.
 
+Note: If there are multiple saga types that handle a given message type and one of them is found while others are not, the `IHandleSagaNotFound` handlers **will not be executed**. The `IHandleSagaNotFound` handlers are only executed if no saga instances are invoked.
```

Comment:
```suggestion
Note: If there are multiple saga types that handle a given message type and one of them is found while others are not, the `IHandleSagaNotFound` handlers **will not be executed**. The `IHandleSagaNotFound` handlers are only executed if no saga instances are invoked. The following table illustrates when the not found handlers are invoked in case when a message is mapped to two different saga types, A and B.

| Saga A found | Saga B found | Not found handler invoked |
|--------|--------|---------|
| ✔️    | ✔️     | ✔️     |
| ✔️    | ❌     | ✔️     |
| ❌    | ❌     | ❌     |
```